app/views/catalog/_citation.html.erb

```diff
@@ -8,6 +8,11 @@
   <% @documents&.each do |document| %>
     <h1 class="modal-title"><%= document_heading(document) %></h1>
 
+    <% if document.respond_to?(:export_as_preferred_citation_txt) %>
+      <h2><%= t('blacklight.citation.preferred') %></h2>
+      <%= document.send(:export_as_preferred_citation_txt).html_safe %><br/><br/>
```

Comment:
why is the `send` needed?

Reply:
No idea, but it's the pattern we've established. I'm happy to kill it, but maybe it's a 8.0 thing?

Reply:
I think `document.export_as_preferred_citation_txt.html_safe` would be idea.